app/src/main/java/org/mozilla/focus/utils/HtmlLoader.java

```diff
@@ -50,47 +47,4 @@ public static String loadResourceFile(@NonNull final Context context,
         }
     }
 
-    private final static byte[] pngHeader = new byte[] { -119, 80, 78, 71, 13, 10, 26, 10 };
-
-    public static String loadPngAsDataURI(@NonNull final Context context,
```

Comment:
We don't need this? Was it used for error pages...?

Reply:
It's used in focus:about: we commented out the code to load the PNG in LocalizedContent.loadAbout. We can re-add it if we need pngs in the about pages.